src/sdk/models/platformCompare/util.ts

```diff
@@ -94,3 +95,30 @@ export const checkPath = (handler: (left: any, right: any) => PlatformComparison
 		return notApplicable();
 	};
 };
+
+export const ifStringArrayExists = (left: any, right: any, key: string): PlatformComparisonRuleResult => {
+	const leftStringArray = left === undefined || left instanceof Array ? left : Array(get(left, key));
+	const rightStringArray = right === undefined || right instanceof Array ? right : Array(get(right, key));
+	if (leftStringArray || rightStringArray) {
+		return {
+			left: leftStringArray || leftStringArray.length !== 0 ? leftStringArray.join(', ') : false,
```

Comment:
Should the CLI be responsible for concatenating the arrays? I would think that would be a UI concern.

Reply:
fixed. removed `join`